spec/features/events_spec.rb

```diff
@@ -79,8 +79,25 @@
       FactoryGirl.create(:application_letter_accepted, :event => @event, :user => @pupil.user)
     end
     visit event_path(@event)
-    click_button I18n.t('events.applicants_overview.sending_acceptances')
-    expect(page).to have_selector('div', :id => 'send-emails-modal')
+    click_link I18n.t('events.applicants_overview.sending_acceptances')
+    fill_in('email_subject', with: 'Subject')
+    fill_in('email_content', with: 'Content')
+    click_button I18n.t('emails.email_form.send')
+  end
```

Comment:
Maybe I am just tired, but what exactly does this test do? It only tests if there is the UI interaction that is needed for sending emails, but whether emails are actually sent is not tested?